pom.xml

```diff
@@ -74,9 +63,47 @@
       <name>Mario Ivankovits</name>
       <email>mario@datenwort.at</email>
     </developer>
+    <developer>
+      <id>madsop-nav</id>
+      <name>Mads Opheim</name>
+      <email>mads.opheim@gmail.com</email>
+    </developer>
   </developers>
 
   <profiles>
+    <profile>
+      <id>github-release</id>
+      <activation>
+        <property>
+          <name>github-release</name>
+        </property>
+      </activation>
+      <distributionManagement>
+        <repository>
+          <id>github</id>
+          <name>GitHub Packages</name>
+          <url>https://maven.pkg.github.com/${env.GITHUB_REPOSITORY}</url>
```

Comment:
Where is the environment variable being set? Is this built in to GitHub Actions?

Reply:
Yep, it is. https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
![image](https://github.com/openhtmltopdf/openhtmltopdf/assets/71336041/fa80347e-77e2-4d9d-a36e-df744a148f27)